ocs_ci/ocs/amq.py

```diff
@@ -446,7 +447,170 @@ def run_in_bg(self, namespace=constants.AMQ_NAMESPACE, value='10000', since_time
 
         return threads
 
-    # ToDo: Install helm and get kafka metrics
+    def run_amq_benchmark(
+        self, name="benchmark", kafka_namespace=constants.AMQ_NAMESPACE,
+        tiller_namespace="tiller", num_of_clients=8, worker=None, timeout=1200,
```

Comment:
Please create a constant for triller in constants.py and use the constant here

Reply:
As discussed, moved at the top of file